rasa_core/train.py

```diff
@@ -111,6 +121,12 @@ def train_dialogue_model(domain_file, stories_file, output_path,
                                                  "augmentation_factor",
                                                  "remove_duplicates",
                                                  "debug_plots"})
+
+    if url is not None:
+        data_store = NamedTemporaryFile()
```

Comment:
please move into a separate function and add error handling (e.g. `response.status_code != 200`)